website/content/docs/job-specification/volume.mdx

```diff
@@ -85,22 +88,39 @@ the [volume_mount][volume_mount] block in the `task` configuration.
   The `per_alloc` field cannot be true for system jobs, sysbatch jobs, or jobs
   that use canaries.
 
-The following fields are only valid for volumes with `type = "csi"`:
+The following fields are only valid for volumes with `type = "csi"` or dynamic
+host volumes with `type = "host"`:
 
-- `access_mode` `(string: <required>)` - Defines whether a volume should be
+- `access_mode` `(string)` - Defines whether a volume should be
   available concurrently. The `access_mode` and `attachment_mode` together
-  must exactly match one of the volume's `capability` blocks. Can be one of
-  `"single-node-reader-only"`, `"single-node-writer"`,
-  `"multi-node-reader-only"`, `"multi-node-single-writer"`, or
-  `"multi-node-multi-writer"`. Most CSI plugins support only single-node
-  modes. Consult the documentation of the storage provider and CSI plugin.
-
-- `attachment_mode` `(string: <required>)` - The storage API that will be used
-  by the volume. The `access_mode` and `attachment_mode` together must exactly
-  match one of the volume's `capability` blocks. Most storage providers will
-  support `"file-system"`, to mount volumes using the CSI filesystem API. Some
-  storage providers will support `"block-device"`, which will mount the volume
-  with the CSI block device API within the container.
+  must exactly match one of the volume's `capability` blocks.
+
+  - For CSI volumes the `access_mode` is required. Can be one of
+    `"single-node-reader-only"`, `"single-node-writer"`,
+    `"multi-node-reader-only"`, `"multi-node-single-writer"`, or
+    `"multi-node-multi-writer"`. Most CSI plugins support only single-node
+    modes. Consult the documentation of the storage provider and CSI plugin.
+
+  - For dynamic host volumes the `access_mode` is optional. Can be one of
+    `"single-node-writer"`, `"single-node-reader-only"`,
+    `"single-node-single-writer"`, or `"single-node-multi-writer"`. Will default
+    to `single-node-writer` unless `read_only = true`, in which case it will
+    default to `single-node-reader-only`.
```

Comment:
```suggestion
  - For dynamic host volumes the `access_mode` is optional. Can be one of the following:
  
    - `"single-node-writer"`
    - `"single-node-reader-only"`
    - `"single-node-single-writer"`
    - `"single-node-multi-writer"` 
    
    Defaults to `single-node-writer` unless `read_only = true`, in which case it
    defaults to `single-node-reader-only`.
```

What do you think of the values in a bullet list so the text is easier to scan?

Reply:
Works for me! :+1: